packages/braintree-utils/src/braintree-sdk-version-manager.ts

```diff
@@ -29,6 +30,10 @@ export default class BraintreeSDKVersionManager {
             return BRAINTREE_SDK_DEFAULT_VERSION;
         }
 
+        if (isExperimentEnabled(features, 'PAYPAL-5809.braintree_hosted_fields_fix_version')) {
```

Comment:
Please move it to line 28, because it should be returned when 
`PAYPAL-5809.braintree_hosted_fields_fix_version` and `PAYPAL-5636.update_braintree_sdk_version` enabled